generation/maisi/maisi_inference_tutorial.ipynb

```diff
@@ -27,15 +27,12 @@
   },
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #27.    mask_generation_diffusion_unet.load_old_state_dict(checkpoint_mask_generation_diffusion_unet)
</pre><p><code>DiffusionModelUNetMaisi</code> don't have <code>load_old_state_dict</code> .</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/1779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
It is DiffusionModelUNet